src/main/java/H2QueryBuilder/H2QueryBuilderDDL.java

```diff
@@ -0,0 +1,103 @@
+    private final static String AUTO_INCREMENT = " AUTO_INCREMENT";
+    private final static String TABLE_NAME     = "{tableName}";
+    private final static String COLUMN_INFO    = "{columnInfo}";
+
+    @Override
+    public String create(Class<?> entityClass) {
+        return generateCreateTableQuery(entityClass);
+    }
+
+    private String generateCreateTableQuery(Class<?> entityClass) {
+        if ( !entityClass.isAnnotationPresent(Entity.class) ) {
+            throw new IllegalArgumentException(ErrorCode.NOT_EXIST_ENTITY_ANNOTATION.getErrorMsg());
+        }
+
+        String columnInfo = getColumn(entityClass)
+                .stream()
+                .map(columnData -> {
+                                        String info = columnData.getColumnName() + " " + columnData.getColumnDataType();
+                                        if (columnData.isNotNull()) info += NOT_NULL;
+                                        if (columnData.isAutoIncrement()) info += AUTO_INCREMENT;
+                                        if (columnData.isPrimeKey()) info += PRIMARY_KEY;
+                                        return info;
+                                    })
+                .collect(Collectors.joining(", "));
+
+        return CREATE_QUERY
+                .replace(TABLE_NAME, getTableName(entityClass))
+                .replace(COLUMN_INFO, columnInfo);
+    }
+
+    // table 이름 가져오기
+    private String getTableName(Class<?> entityClass) {
+        if (entityClass.isAnnotationPresent(Table.class)) {
+            Table table = entityClass.getAnnotation(Table.class);
+            return table.name();
+        }
+        return entityClass.getSimpleName();
+    }
+
+    //조립된 컬럼 가져오기
+    private List<ColumnData> getColumn(Class<?> entityClass) {
+        List<ColumnData> columnDatas = new ArrayList<>();
+
+        Field[] fields = entityClass.getDeclaredFields();
+        Arrays.stream(fields).forEach(field -> generateTableColumnData(columnDatas, field));
+
+        return columnDatas;
+    }
+
+    // column 조립
+    private void generateTableColumnData(List<ColumnData> columnDatas, Field field) {
+        String columnName = field.getName();
+        boolean isNullable = true;
+
+        if ( field.isAnnotationPresent(Column.class) ) {
+            Column column = field.getAnnotation(Column.class);
+            columnName  = column.name().isEmpty() ? columnName : column.name();
+            isNullable = column.nullable();
+
+        }
+
+        if ( field.isAnnotationPresent(Id.class) ) {
+            columnDatas.add(new ColumnData(columnName, field.getType(), true, true, isGenerateValueIdentity(field)));
+        } else {
+            columnDatas.add(new ColumnData(columnName, field.getType(), false, !isNullable, isGenerateValueIdentity(field)));
+        }
+    }
+
+    // GeneratedType 체크
+    private boolean isGenerateValueIdentity(Field field) {
+        if ( !field.isAnnotationPresent(GeneratedValue.class) ) {
```

Comment:
```suggestion
        if (!field.isAnnotationPresent(GeneratedValue.class)) {
```
👀 
intellij에서는 ⌥⌘L 또는 Ctrl+Alt+L 로 코드 정렬을 제공하고 있으니 습관적으로 눌러주시는 것도 좋을 것 같아요 :)

Reply:
아 이건 지금 회사 팀에서 이렇게 룰을 정해서,, 습관이 되었네요..

Reply:
해당 미션은 일반적으로 google java convention을 적용하고 있어 다른 미션에서 리뷰를 받을 수도 있는점 미리 참고해주시면 좋을 것 같아요.